### PR TITLE
Do not use instructions unsupported by Z Arch level

### PR DESCRIPTION
**Pull request by dhong44**
Do not use instructions unsupported by the Z architecture level and 
add testing for `-Xjit:disable<Arch>` for Z architecture levels.

Closes: #3577

### CODE REVIEW

**Comment by dhong44** [comment]
@fjeremic

**Comment by dhong44** [comment]
Marking as WIP until my testing is complete.

**Comment by dhong44** [comment]
@fjeremic Could I get a review on this please? it is ready now.

File: test/functional/cmdLineTests/cmdLineTest_J9tests/j9tests_Java8.xml [hunk 1]
```diff
@@ -513,12 +513,32 @@
...
+
+ <test id="Test option -Xjit:count=0,disableZ13">
+  <command>$EXE$ -Xjit:count=0,disableZ13 -version</command>
+  <output regex="no" type="success">version</output>
+ </test>
+
+ <test id="Test option -Xjit:count=0,disableZ14">
+  <command>$EXE$ -Xjit:count=0,disableZ14 -version</command>
+  <output regex="no" type="success">version</output>
+ </test>
```
Comment (fjeremic) [inline test/functional/cmdLineTests/cmdLineTest_J9tests/j9tests_Java8.xml]: Let's also add a `disableZ15` test as well.

Comment (dhong44) [reply test/functional/cmdLineTests/cmdLineTest_J9tests/j9tests_Java8.xml]: Added in 0a6890d

**Comment by fjeremic** [comment]
Jenkins test sanity zlinux JDK8 depends eclipse/omr#3659

**Pull request merged**